Source/WebCore/css/parser/CSSParserContext.h

```diff
@@ -109,7 +109,7 @@ struct CSSParserContext {
 
     CSSParserContext(CSSParserMode, const URL& baseURL = URL());
     WEBCORE_EXPORT CSSParserContext(const Document&);
-    CSSParserContext(const Document&, const URL& baseURL, const String& charset = emptyString());
+    CSSParserContext(const Document&, const URL& baseURL, ASCIILiteral charset = ASCIILiteral());
```

Comment:
I don't know if it matters but this is not equivalent to the old code. The equivalent would be `ASCIILiteral charset = ""_s`

Reply:
Ah! Good catch. I will fix that. Maybe that is the source of my test failures.